app/seqexec-server-gs/src/main/resources/app.conf

```diff
@@ -61,7 +61,7 @@ seqexec-engine {
   epicsCaAddrList = "172.17.2.255 172.17.3.255 172.17.102.130 172.17.105.20 172.16.102.130 172.17.106.111 172.17.105.37 172.17.107.50 172.17.55.101 172.17.101.101 172.17.65.255 172.17.102.139 172.17.102.138"
   readRetries = 4
   ioTimeout = 4 seconds
-  dhsTimeout = 24 seconds
+  dhsTimeout = 30 seconds
```

Comment:
This is an unrelated change. This timeout was increased manually months ago because of some problems with the DHS.